pubspec.yaml

```diff
@@ -122,7 +122,9 @@ dependencies:
   # enough_html_editor: Compose email for Mobile & Tablet
   enough_html_editor:
     git:
-      url: https://github.com/Enough-Software/enough_html_editor.git
+#      url: https://github.com/Enough-Software/enough_html_editor.git
+      url: https://github.com/dab246/enough_html_editor.git
```

Comment:
Should be a linagora repo IMO.

We also need a link to a ticket as a comment explaining why we need that...

Reply:
he did, please check in the description

Reply:
Yes but imo we deserve a way to find back this info from source repo. Ie with a comnent linking to this pr

Reply:
My PR was merged into the master branch by them, so this bug has been fixed on our project. We can close this PR without merging it. @hoangdat @chibenwa  
[https://github.com/Enough-Software/enough_html_editor/pull/30](https://github.com/Enough-Software/enough_html_editor/pull/30)